packages/server/src/api/controllers/apikeys.js

```diff
@@ -0,0 +1,46 @@
+const fs = require("fs")
+
+const ENV_FILE_PATH = ".budibase/.env"
```

Comment:
There's a helper for this in `utilities/budibaseDir` that will work cross platform

Reply:
Neat! Switched it out. Cheers!